### PR TITLE
Replace classnames with clsx

### PR DESCRIPTION
**Pull request by cahna**
See: 
- https://material-ui.com/getting-started/faq/#whats-the-clsx-dependency-for
- https://github.com/mui-org/material-ui/pull/14152

### CODE REVIEW

**Comment by cahna** [comment]
I may become a regular around here. After extensive research, this seems to be the best available table component for my needs. I would be happy to help-out on weekends with any small tasks/busywork you may want to delegate.
I can also help with an iterative migration to TypeScript and/or Jest+@testing-library/react if they are desired. Or if there is any desire to add cypress tests (ie: for DnD coverage), I can help there.

**Comment by patorjk** [comment]
I'm pretty open to ideas. I'm not really familiar enough with TypeScript to want to port over, though anything to make it more accessible to TypeScript users would be nice. Right now I believe most TypeScript users use the DefinitelyTyped project with this project. Not sure if there's a better way support could be offered.

Tests for DnD would be great! Don't feel obligated though. Anything you want to contribute would be appreciated.

**Comment by cahna** [comment]
A preview of DnD testing (https://github.com/cahna/mui-datatables/pull/1):
draggable-columns spec js

File: package.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
     "react-dom": "^16.8.0"
   },
   "dependencies": {
-    "classnames": "^2.2.6",
+    "clsx": "^1.1.1",
```
Comment (wdh2100) [inline package.json:100]: #1415 Reason

**Pull request merged**